app/models.py

```diff
@@ -3,71 +3,71 @@
 
 from sqlalchemy import create_engine
 from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.orm import relationship, backref
+from sqlalchemy.exc import OperationalError
 import os
-
-
 """ DO NOT COMMIT CHANGES TO THIS FILE!! """
 try:
- 	engine
+    engine
 except NameError:
-	pass
+    pass
 else:
-	engine.dispose()
-	print("~~~~~~~~~~~~~connection closed~~~~~~~~~~~~~~~~~")
+    engine.dispose()
+    print("~~~~~~~~~~~~~connection closed~~~~~~~~~~~~~~~~~")
 
 if 'MYSQL_DATABASE_URL' in os.environ and os.environ['MYSQL_DATABASE_URL']:
     db_url = os.environ['MYSQL_DATABASE_URL']
 else:
     db_url = 'mysql+pymysql://root:@127.0.0.1:3306/youtubelib'
 
-engine = create_engine(db_url+'?charset=utf8', convert_unicode=True, echo=False)
-print ("~~~~~~~~~~~~~~connection opened~~~~~~~~~~~~~~~~~~~~~~")
+engine = create_engine(
+    db_url + '?charset=utf8', convert_unicode=True, echo=False)
+print("~~~~~~~~~~~~~~connection opened~~~~~~~~~~~~~~~~~~~~~~")
 
 Base = declarative_base()
-Base.metadata.reflect(engine)
-
-class Album(Base):
-	__table__ = Base.metadata.tables['albums']
-
-class Artist(Base):
-	__table__ = Base.metadata.tables['artists']
+try:
+    Base.metadata.reflect(engine)
```

Comment:
This line throws an exception if there's no database available. Since this is the first module imported, it means that if the database isn't running, the rest of the code doesn't even get imported. I would strongly advising organizing your code in such a way that as much of the application as possible still works without there being a database connection.a